src/main/groovy/com/palantir/configurationresolver/ResolveConfigurationsTask.groovy

```diff
@@ -22,6 +22,12 @@ public class ResolveConfigurationsTask extends AbstractTask {
     @TaskAction
     public void resolveAllConfigurations() {
         project.configurations.all { configuration ->
+            // New versions of Gradle have unresolvable configurations by default
+            // https://github.com/gradle/gradle/pull/1351
+            if (configuration.metaClass.respondsTo(configuration, "isCanBeResolved") &&
+                !configuration.isCanBeResolved()) {
+              return;
```

Comment:
I think it's fine for it to fail quietly. These "unresolvable configurations" are going to automatically be added to every project by Gradle itself starting in 3.4, so logging would produce a lot of noise and almost no signal.